src/client.cc

```
@@ -134,60 +134,101 @@ struct ScopedRDMARegistration {
 }
 
-void ListObjectsResult::Populate() {
+void ListObjectsResult::UpdatePaginationArgs() {
   if (args_.include_versions) {
-    args_.key_marker = resp_.next_key_marker;
-    args_.version_id_marker = resp_.next_version_id_marker;
+    args_.key_marker = resp_->next_key_marker;
+    args_.version_id_marker = resp_->next_version_id_marker;
   } else if (args_.use_api_v1) {
-    args_.marker = resp_.next_marker;
+    args_.marker = resp_->next_marker;
```

Cppcheck (2.21.0)

[style] The function 'JsonTypeToString' is never used.

(unusedFunction)